src/Microsoft.Identity.Client/IPublicClientApplication.cs

```diff
@@ -59,15 +60,15 @@ public partial interface IPublicClientApplication : IClientApplicationBase
         /// <remarks>This API will be removed in MSAL v3.x. See https://aka.ms/msal-net-ios-keychain-security-group for details</remarks>
         [Obsolete("Use iOSKeychainSecurityGroup instead (See https://aka.ms/msal-net-ios-keychain-security-group)", false)]
         string KeychainSecurityGroup { get; set; }
-#endif
+#endif // iOS
```

Comment:
we were going to remove this property in v3...do you want to do it now or wait?

Reply:
Let's hold and do this explicitly.  I don't want to introduce other changes with these large PRs because it's hard to find where we explicitly removed something like this (if we need to add it back).

Reply:
cool. sounds like a plan. maybe add that to your list of things to do (so we don't forget). 

---
In reply to: [247296973](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/781#discussion_r247296973) [](ancestors = 247296973)